### DISCUSSION TITLE
We created Edge, an OpenSource tool on top of NextJS (dynamic API, and dynamic dashboard)

### DISCUSSION THREAD

**Discussion by rafinskipg** [vercel/next.js#13633] [Show and tell]
Hello, friends. We want to share this tool we created using NextJS and gather feedback from the community to see what to change, what to improve, what is right, what is wrong. 

What does it offer?
- Integrated dynamic API and dashboard
- Roles for the API and the client side
- Content types with dynamic fields and dynamic forms
- Comments and discussions
- Web monetization layer (on the client side)
- Set of coded components

We wrote a post on Dev.to to cover the launch and also on Edge. Here you can find more about what the project is about and all the features. 

[Dev To post](https://dev.to/rafinskipg/edge-an-opensource-nextjs-application-with-dynamic-api-for-makers-and-developers-313)
[Post on Edge demo site](https://edge-next.now.sh/content/post/what-is-edge-1590913556408)

Some feedback we already gathered from the community:
- change the name
- use typescript

We would like to hear your ideas and critics, 

thanks!

**Comment by Vadorequest** [comment]
Interesting, I'll take a look to see what I can re-use in https://github.com/UnlyEd/next-right-now 👍 

Don't hesitate to take a look at it, especially if you're looking for some doc on Next.js with TS 😉

**Reply by rafinskipg** [reply]
hey @Vadorequest , also if you want to collaborate with us, go ahead

**Reply by Vadorequest** [reply]
Edge provides interesting features, it's very different from NRN, though. 
But it's definitely interesting, I bookmarked in my "MVP" folder 😉

**Reply by Vadorequest** [reply]
Added to NRN own list of alternatives/inspirations https://github.com/UnlyEd/next-right-now/issues/87 😉

**Comment by rafinskipg** [comment]
Adding documentation on how to create groups, for the next rewrite of the code: https://github.com/nucleo-org/edge-next/blob/groups/DOCUMENTATION.md#groups
